src/types.ts

```diff
@@ -11,6 +11,15 @@ export interface ProxyReaderMap {
 export type NetworkIdMap = {
   [key: number]: string;
 };
+
+export type ProxyData = {
+  owner?: string,
+  resolver?: string,
+  values?: string[]
```

Comment:
Proxy reader will nerver return `null` or `undefined` or `0x`. It will always be `0x000000000...000`:

```suggestion
  owner: string,
  resolver: string,
  values: string[]
```

Additional `VerifiedData` will not be needed in this case.